app/src/main/java/org/gdg/frisbee/android/fragment/LeadFragment.java

```diff
@@ -0,0 +1,94 @@
+package org.gdg.frisbee.android.fragment;
+
+import android.app.Activity;
+import android.content.Context;
+import android.content.Intent;
+import android.net.Uri;
+import android.os.Bundle;
+import android.support.annotation.Nullable;
+import android.support.v4.app.Fragment;
+import android.support.v4.app.ListFragment;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.ArrayAdapter;
+import android.widget.ImageView;
+import android.widget.ListView;
+import android.widget.TextView;
+
+import org.gdg.frisbee.android.Const;
+import org.gdg.frisbee.android.R;
+import org.gdg.frisbee.android.lead.LeadMessage;
+
+public class LeadFragment extends ListFragment {
+    private LeadAnouncementAdapter mAdapter;
+
+    @Override
+    public View onCreateView(final LayoutInflater inflater, @Nullable final ViewGroup container, @Nullable final Bundle savedInstanceState) {
+        View root = inflater.inflate(R.layout.fragment_for_leads, null);
+        return root;
+    }
+
+    public static Fragment newInstance(final String gplusId) {
+        Fragment fragment = new LeadFragment();
+        Bundle arguments = new Bundle();
+        arguments.putString(Const.EXTRA_PLUS_ID, gplusId);
+        fragment.setArguments(arguments);
+        return fragment;
+    }
+
+    @Override
+    public void onActivityCreated(Bundle savedInstanceState) {
+        super.onActivityCreated(savedInstanceState);
+        mAdapter = new LeadAnouncementAdapter(getActivity());
+        setListAdapter(mAdapter);
+        mAdapter.add(LeadMessage.newMessage(getString(R.string.leads_welcome_title),
+                getString(R.string.leads_welcome)));
+        mAdapter.add(LeadMessage.newResource(getString(R.string.leads_resources_title),
+                getString(R.string.leads_resources),
+                "https://drive.google.com/drive/#folders/0B55wxScz_BJtWW9aUnk2LUlNdEk"));
```

Comment:
I know. But for now (if we are going to merge it) we shouldn't have duplicate String constants.